loadtests/aad-load-testing/config.yaml

```diff
@@ -0,0 +1,33 @@
+displayName: Contoso traders with AAD
+testPlan: Contoso-traders-aad.jmx
+description: CICD test from GitHub Actions
+engineInstances: 2
+testId: contosotradersaad
+splitAllCSVs: False
+configurationFiles:
+- Samsung phone.jpg
+failureCriteria:
+- View product: percentage(error) > 10
+- Product landing page: p90(response_time_ms) > 3000
+- View cart: p90(response_time_ms) > 100
+secrets:
+- name: clientId
+  value: https://contosotradersloadtest.vault.azure.net/secrets/clientId
```

Comment:
The suffixes might vary (per the `SUFFIX` github repository secret). We shouldn't assume that it'll always be `test`.